src/main/java/no/finn/unleash/util/UnleashConfig.java

```diff
@@ -88,8 +101,17 @@ public UnleashContextProvider getContextProvider() {
         return contextProvider;
     }
 
+    public static void setRequestProperties(HttpURLConnection connection, UnleashConfig config) {
+        connection.setRequestProperty(UNLEASH_APP_NAME_HEADER, config.getAppName());
+        connection.setRequestProperty(UNLEASH_INSTANCE_ID_HEADER, config.getInstanceId());
```

Comment:
Sure. I didn't think to add it as there was no `User-Agent` in the code to begin with, but I'll add it now. Based on the linked commit, I take it the `User-Agent` should be the app name.